rocky/rocky/templates/header.html

```diff
@@ -50,6 +51,14 @@
                                 <a href="{{ index_url }}"
                                    {% if index_url in request.path|urlencode %}aria-current="page"{% endif %}>{% translate "Findings" %}</a>
                             </li>
+                            {% is_feature_reports as feature_reports %}
+                            {% if feature_reports %}
```

Comment:
Not really a problem, but I don't understand the `is_feature_reports as feature_reports`. Why not just call the template tag `feature_reports`?